### DISCUSSION TITLE
For input value loss

### DISCUSSION THREAD

**Discussion by wangshiliang888** [solidjs/solid#2053] [Q&A]
const selectedInput = () => {
  const [value, setValue] = createSignal("123456");
  return (
    <For each={shareStore.selected}>
      {(item, i) => {
        return (
          <div class="selected-item" key={item.id}>
            {item.title} [{item.price}] {item.id} x
            <input type="number" name={`products-${i()}-pk`} value={value()} />
            <input type="text" name={`products-${i()}-title`} value={value()} />
            <input type="number" name={`products-${i()}-price`} value={value()} />
          </div>
        );
      }}
    </For>
  );
};

![image](https://github.com/solidjs/solid/assets/146353603/0ef33067-686e-4470-aac6-ad4502f0693b)

**The value is lost after rendering the above code to HTML**

**Comment by Daniel-Khodabakhsh** [comment]
Hi @wangshiliang888 
I believe you should look here for some answers to your question: https://github.com/solidjs/solid/discussions/416

My understanding is that what you're trying to do is very specific to React and isn't done in Solid (and most other frameworks).
